examples/rhoai-quickstart/feast-demo-quickstart.ipynb

```diff
@@ -0,0 +1,1530 @@
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "pd.read_parquet(\"data/driver_stats.parquet\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "92557e68-f044-444e-a68a-31385b95092e",
+   "metadata": {},
+   "source": [
+    "You have not created any feast objects to do that you have to execute command `feast apply` on the directory where `feature_store.yaml` exists. Lets go and do that now."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "8e0a05ec-f449-4371-9d55-f30176b51b38",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Below folder is creating interference with the feast apply command so deleting it in case if it exists.\n",
+    "!rm -rf .ipynb_checkpoints/"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "5c6baa83-9834-46ac-82a4-cc0a9ae33080",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/opt/app-root/lib64/python3.9/site-packages/feast/feature_store.py:590: RuntimeWarning: On demand feature view is an experimental feature. This API is stable, but the functionality does not scale well for offline retrieval\n",
+      "  warnings.warn(\n",
+      "09/24/2024 03:08:32 AM root WARNING: Cannot use sqlite_vec for vector search\n",
+      "09/24/2024 03:08:32 AM root WARNING: Cannot use sqlite_vec for vector search\n",
+      "09/24/2024 03:08:32 AM root WARNING: Cannot use sqlite_vec for vector search\n",
+      "09/24/2024 03:08:32 AM root WARNING: Cannot use sqlite_vec for vector search\n",
+      "Created entity \u001B[1m\u001B[32mdriver\u001B[0m\n",
+      "Created feature view \u001B[1m\u001B[32mdriver_hourly_stats\u001B[0m\n",
+      "Created feature view \u001B[1m\u001B[32mdriver_hourly_stats_fresh\u001B[0m\n",
+      "Created on demand feature view \u001B[1m\u001B[32mtransformed_conv_rate\u001B[0m\n",
+      "Created on demand feature view \u001B[1m\u001B[32mtransformed_conv_rate_fresh\u001B[0m\n",
+      "Created feature service \u001B[1m\u001B[32mdriver_activity_v2\u001B[0m\n",
+      "Created feature service \u001B[1m\u001B[32mdriver_activity_v1\u001B[0m\n",
+      "Created feature service \u001B[1m\u001B[32mdriver_activity_v3\u001B[0m\n",
+      "\n",
+      "09/24/2024 03:08:32 AM root WARNING: Cannot use sqlite_vec for vector search\n",
+      "09/24/2024 03:08:32 AM root WARNING: Cannot use sqlite_vec for vector search\n",
+      "Created sqlite table \u001B[1m\u001B[32mmy_feast_project_driver_hourly_stats_fresh\u001B[0m\n",
+      "Created sqlite table \u001B[1m\u001B[32mmy_feast_project_driver_hourly_stats\u001B[0m\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "# this command will actual creates the feast objects mentioned in `example_repo.py`\n",
+    "!feast apply"
```

Comment:
I think we miss sub-sections like "Create a sample feast project" and "Initialize the feast registry" or similar.

Reply:
Added the sub section now.